### PR TITLE
fix: input style in API design

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
image
image

## Does this PR introduce a user interface change?
- [x] Yes(screenshot is required)
- [ ] No

## Which versions should be patched?
master & release/1.1

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/styles/antd-extension.scss [hunk 1]
```diff
@@ -326,10 +326,6 @@ input.ant-input {
   display: inline-flex;
 }
 
-.api-filter-input {
-  width: auto;
-}
```
Comment (daskyrk) [inline shell/app/styles/antd-extension.scss:331]: will this deletion affect other place?

Comment (sherotree) [reply shell/app/styles/antd-extension.scss:331]: > will this deletion affect other place?

image
only one place use the className

**Pull request merged**